app/jobs/send_complaint_job.rb

```diff
@@ -19,6 +21,8 @@ def perform(form_builder_payload:)
       get_bearer_token: bearer_token
     ).execute
 
+    Rails.logger.info("Sent #{form_builder_payload[:submissionId]} to Optics")
```

Comment:
Are we ok always logging/exposing the submission ID? Suggest this could be a DEGBUG level log